cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -933,6 +954,13 @@ func init() {
 	createCmd.Flags().IntVar(&controlPlanePort, controlPlanePortFlag, constants.DefaultControlPlanePort, "control plane port (load balancer and local API port)")
 	createCmd.Flags().BoolVar(&dhcpSkipHostname, "disable-dhcp-hostname", false, "skip announcing hostname via DHCP (QEMU only)")
 	createCmd.Flags().BoolVar(&skipBootPhaseFinishedCheck, "skip-boot-phase-finished-check", false, "skip waiting for node to finish boot phase")
+	createCmd.Flags().BoolVar(&networkChaos, networkChaosFlag, false, "enable to use network chaos parameters when creating a qemu cluster")
+	createCmd.Flags().IntVar(&jitter, jitterFlag, 0, "specify jitter (in ms) on the bridge interface when creating a qemu cluster (default: 0)")
+	createCmd.Flags().IntVar(&latency, latencyFlag, 0, "specify latency (in ms) on the bridge interface when creating a qemu cluster (default: 0)")
```

Comment:
in Go, every time-related thing is better represented as `time.Duration`, and the flag of the same type